### PR TITLE
docs:updated docs so that the selector param description is consistent

### PR DESCRIPTION
**Pull request by camfung**
**What kind of change does this PR introduce?**
Docs update so that the description for selector is consistent with other descriptions for it

**Did you add tests for your changes?**
No

**Summary**
I noticed that the selector didn't link to the docs on mdn so i added it to the Page.click docs. 

I wanted to add this because when i was looking through the docs for clicking and the selector seamed ambiguous to me.

**Does this PR introduce a breaking change?**
no

### CODE REVIEW

**Review by OrKoN — CHANGES_REQUESTED** [review]
could you please update the comment in the source .ts file and run `npm run docs` to regenerated the markdown files? thanks!

File: packages/puppeteer-core/src/api/Page.ts [hunk 1]
```diff
@@ -2672,8 +2672,10 @@ export abstract class Page extends EventEmitter<PageEvents> {
    * ```
    *
    * Shortcut for {@link Frame.click | page.mainFrame().click(selector[, options]) }.
-   * @param selector - A `selector` to search for element to click. If there are
-   * multiple elements satisfying the `selector`, the first will be clicked
+   * @param selector - A
+   * {@link https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors | selector}
+   * of an element to type into. If there are multiple elements satisfying the
```
Comment (OrKoN) [inline packages/puppeteer-core/src/api/Page.ts]: I think this should say "to click" instead of "to type into"?

Comment (camfung) [reply packages/puppeteer-core/src/api/Page.ts]: Shoot sorry thats a dumb typo

**Comment by OrKoN** [comment]
Could you please sign the CLA as requested by the CLA check?

**Pull request closed without merging**